server/routes/templates.ts

```diff
@@ -133,7 +133,8 @@ templateRoutes.route("/").get(authenticateWithRedirect, async (request, response
 		user: request.user,
 		settings: {
 			teamsEnabled: await getSetting<boolean>("teamsEnabled"),
-			qrEnabled: await getSetting<boolean>("qrEnabled")
+			qrEnabled: await getSetting<boolean>("qrEnabled"),
+			exportKey: await getSetting<string>("exportKey")
```

Comment:
Why is the export key being sent to the template engine for routes that don't need it?